assignment-2/7s.md

```diff
@@ -1,2 +1,296 @@
+> perform their jobs. Leadership is included in this category, whether leaders are selected from the
+> outside or developed from within. Staffing addresses the question of how the organization
+> assures it has the right people to do the work.
+
+Spotify claims to have a very clear definition of what and how employees careers should develop in order to achieve the most important organization goals.[6]"Everyone in *Spotify tech* should have a way to grow their careers and expand the impact of their -work no matter what role they play".
+The Spotify technology career framework, consist on letting the employee transfer itself on as many roles and domain expertise as possible in order to gain experience and later evaluate their impact into the organization.
+*On recognition and promotions* Spotify observes constantly the impact someone has at a wider scope, and they have a formal ways to promote their employees.
+*On setting the step for a new employee* Spotify gives 6 months to all new employees to get involved with their role and the organization.
+*On compensation* A formal promotion should lead to higher pay in a straightforward and transparent way.
+
+##### Skills
+
+> Skills are the special, distinctive competencies of the organization, both in the people and the
+> organization itself. What is the organization good at, and what special skills and competencies
+> are found its people?
+
+What is the organization good at?
+According to [17]:
+- Spotify has better social features than its rivals, with the ability to add friends and see their playlists and listening, as well as a built-in messaging system to ping music back and forth. Creating and sharing your own playlists is easy too.
+- Another strength is the system for following musicians’ profiles and getting notifications when they have new music out. A recently introduced Concerts feed shows you upcoming, nearby gigs by artists you’ve listened to, which is useful.
+- It has good support for devices beyond the desktop and mobile: it’s available on Sony’s PlayStation 4, Sonos hi-fis and through Google’s Chromecast and Amazon’s Fire TV, while its mobile app plays nicely with Apple TV too.
+- Spotify’s free tier has audio advertising, but is missing offline listening and – on mobile – the ability to pick tracks on-demand, although you can play an artist’s catalogue on shuffle mode. Pay £9.99 a month, and you get full access (including offline), no ads, and a higher-quality streams option.
+- Where Spotify falls down is its family plan, which is more expensive than rivals. You can add up to four extra accounts for £4.99 each, but that means you’ll pay £29.95 a month for a family of five compared to Apple Music and Google Play’s £14.99 for six.
+- Besides all this facts actually there are a couple of easy things to say where Spotify is very good at, like Spotify seems to be good at keeping their organization dynamic, focusing on their community rather to an hierarchical structure.
```

Comment:
"Besides all this facts actually there are a" => "Besides all of these, there actually are a"